MFVNeutralino/plugins/MiniTreer.cc

```diff
@@ -84,6 +85,7 @@ void MFVMiniTreer::analyze(const edm::Event& event, const edm::EventSetup&) {
   nt.pvx = mevent->pvx - mevent->bsx_at_z(mevent->pvz);
   nt.pvy = mevent->pvy - mevent->bsy_at_z(mevent->pvz);
   nt.pvz = mevent->pvz - mevent->bsz;
+  if (mevent->npu<0 || mevent->npu>255) return;
```

Comment:
Remind me, how often did you find this happening?

Reply:
I don't really remember adding this in code. Maybe it comes from my merging some upstream changes in the original code. I haven't encountered those case before.

Reply:
Hmm, I vaguely remember this but don't see it in the master branch. Maybe it was in one of my other branches...

Well, if the npu is actually > 255, it would be a bit hard for us to use, so probably fine as is.